geode-core/src/main/java/org/apache/geode/distributed/internal/membership/gms/mgr/GMSMembershipManager.java

```diff
@@ -2364,8 +2364,7 @@ public void unregisterTestHook(MembershipTestHook mth) {
         if (this.membershipTestHooks.size() == 1) {
           this.membershipTestHooks = null;
         } else {
-          List<MembershipTestHook> l = new ArrayList<>(this.membershipTestHooks);
-          l.remove(mth);
```

Comment:
Bruce is right. Compare this block to line 2353.

Reply:
fixed it. Thank you